src/components/DatePicker/DatePicker.jsx

```diff
@@ -182,32 +204,67 @@ export const DatePicker = ({
                     </div>
                 ) : null}
 
-                <DayPicker
-                    className={clsx(
-                        "ui-date-picker rounded-lg pt-3",
-                        useDateRangeStyle ? "date-range-picker" : null,
-                        getDayContent ? "has-custom-content" : null,
-                        modifiers.waitlist ? "has-custom-content" : null,
-                    )}
-                    todayButton={variant === "single" ? "Today" : undefined}
-                    selectedDays={selectedDays}
-                    month={currentMonth}
-                    modifiers={{ ...modifiers, ...rangeModifier }}
-                    numberOfMonths={isRangeVariant ? 2 : 1}
-                    disabledDays={disabledDays}
-                    captionElement={captionElement}
-                    renderDay={renderDay}
-                    navbarElement={NavbarElement}
-                    onDayClick={handleDayClick}
-                    onMonthChange={handleMonthChange}
-                    onTodayButtonClick={handleTodayClick}
-                    {...rest}
-                />
+                {isRangeVariant ? (
+                    <div className="flex gap-4">
+                        <CustomDayPicker
+                            todayButton={variant === "single" ? "Today" : undefined}
+                            isRangeVariant={isRangeVariant}
+                            isDateRangeStyle={isDateRangeStyle}
+                            month={startMonth}
+                            modifiers={{ ...modifiers, ...rangeModifier }}
+                            disabledDays={disabledStartDays}
+                            captionElement={captionStartElement}
+                            getTooltip={getTooltip}
+                            getDayContent={getDayContent}
+                            value={value}
+                            isDisabled={isDisabled}
+                            onDayClick={handleDayClick}
+                            onMonthChange={handleStartMonthChange}
+                            onTodayButtonClick={handleTodayClick}
+                            {...rest}
```

Comment:
this still have as many props as original one. can you create a component `<RangeDatePicker />` and internally can use the original DayPicker itself. That will avoid declaring the type for this CustomDayPicker again